### DISCUSSION TITLE
Right Click Properties Missing

### DISCUSSION THREAD

**Discussion by ligistx** [deepskystacker/DSS#177] [Ideas]
I have historically had some strange issues with my intervalometer being exactly 180 seconds as seen in this example (I think I have that fixed moving forward...), but to fix this I select many images and edit all of their exposure times as shown in the image below. This seems to be missing from 5.1.0 which is a somewhat game breaking issue for me and likely others. My darks are exactly 180 seconds, so I need a watch to batch adjust the exposure time so the darks apply correctly.

![image](https://user-images.githubusercontent.com/44333637/225509338-efeb499b-801f-419f-9b58-af05f1f2513a.png)

**Comment by perdrix52** [comment]
You're not the first to ask I will look to reinstate that in 5.1.1 (soon)

**Reply by ligistx** [reply]
That would be awesome!

Love the program, and very happy to see it get a facelift and overhaul :).
